marshal.go

```diff
@@ -333,7 +333,7 @@ func marshalVarchar(info TypeInfo, value interface{}) ([]byte, error) {
 	case k == reflect.Slice && t.Elem().Kind() == reflect.Uint8:
 		return rv.Bytes(), nil
 	}
-	return nil, marshalErrorf("can not marshal %T into %s", value, info)
+	return nil, marshalErrorf("can not marshal %T into %s. Accepted types: Marshaler, string, []byte, unsetColumn.", value, info)
```

Comment:
`unsetColumn` is not an exported type, so the driver doesn't have to expose it to the user. There is a `UnsetValue` global var of type `unsetColumn` which is meant to be used to ignore writing. You can find it [here](https://github.com/apache/cassandra-gocql-driver/blob/2a5056d6b1d1d479ca5ee97ddd1d95a9d8e8c4e2/frame.go#L50).

Reply:
Thank you, refactored

Reply:
I meant to change unsetColumn to UnsetValue, not to delete it at all

Reply:
I see, fixed.